tests/testpyipv8packets.rs

```diff
@@ -1,151 +1,151 @@
-  let mut iter = data.deserialize_multiple();
-
-  assert_eq!(
-    BinMemberAuthenticationPayload {
-      public_key_bin: VarLen16(
-        vec![76, 105, 98, 78, 97, 67, 76, 80, 75, 58, 81, 231, 18, 196, 235, 138, 194, 90, 227, 165, 104, 36, 8, 178, 173, 189, 107, 120, 164, 37, 84, 127, 38, 133, 207, 223, 30, 233, 39, 12, 190, 126, 195, 54, 196, 22, 15, 245, 114, 5, 76, 135, 120, 66, 190, 55, 115, 80, 69, 169, 59, 196, 226, 4, 21, 49, 111, 219, 20, 113, 97, 162, 215, 70]
-      )
-    },
-    iter.next().unwrap()
-  );
-
-  assert_eq!(
-    TimeDistributionPayload{
-      global_time: 151,
-    },
-    iter.next().unwrap()
-  );
-
-  // tmp to verify the introduction request payload. Maybe useful for verifying the todos below.
-  // let p: IntroductionRequestPayload = iter.next().unwrap();
-  // println!("{:?}",p);
-
-  assert_eq!(
-    IntroductionRequestPayload {
-
-      destination_address: Address {
-        address: Ipv4Addr::new(81, 171, 27, 194),
-        port: 11111
-      },
-      source_lan_address: Address {
-        address: Ipv4Addr::new(192,168,1,75),
-        port: 8090
-      },
-      source_wan_address: Address {
-        address: Ipv4Addr::new(192,168,1,75),
-        port: 8090
-      },
-      // TODO i am not entirely sure of the extraction of bits is 100% right. though public does seem plausible.
-      advice: false,
-      connection_type: ConnectionType::PUBLIC,
-      identifier:151,
-      // TODO: i have not at all verified if this RawEnd is correct yet.
-      extra_bytes: RawEnd(
-        vec![0, 0, 0, 0, 3, 165, 219, 245, 204, 2, 26, 33, 233, 112, 45, 138, 242, 145, 180, 98, 73, 145, 178, 65, 8, 137, 227, 69, 161, 156, 47, 249, 11, 96, 38, 201, 112, 104, 42, 188, 209, 3, 71, 45, 253, 238, 25, 216, 249, 72, 110, 191, 43, 253, 231, 12, 134, 215, 188, 0, 166, 33, 254, 38, 34, 137, 218, 11]
-      )
-    },
-    iter.next().unwrap()
-  )
-}
+//use ipv8::networking::serialization::Packet;
```

Comment:
Please either delete this file or uncomment it.

Reply:
no, we are fixing it in the next PR. just leave it for now